backend/test_jina_integration.py

```
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+"""
+测试Jina API集成功能的脚本
+"""
+
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)